server/api/pipeline.go

```diff
@@ -87,8 +87,7 @@ func createTmpPipeline(event model.WebhookEvent, commitSHA string, repo *model.R
 		Author: user.Login,
 		Email:  user.Email,
 
-		// TODO: Generate proper URL to commit
-		ForgeURL: repo.ForgeURL,
+		ForgeURL: commit.ForgeURL,
```

Comment:
Should we maybe rename this to `ForgeContextUrl` as its not just the ForgeUrl, but the url to a PR / commit, tag, ...?